qualification/tied_array_channelised_voltage/test_delay.py

```diff
@@ -186,7 +188,8 @@ async def test_delay(
 
         corr_phase = np.angle(corr)
         fig = Figure(tight_layout=True)
-        ax, ax_err = fig.subplots(2)
+        # matplotlib's typing doesn't specialise for Nx1 case
+        ax, ax_err = cast(Sequence[Axes], fig.subplots(2))
```

Comment:
If this is a common pattern it may be worth filing a bug / feature request for this particular issue?

I can forsee this code pattern getting boring pretty quickly.

Reply:
You mean like https://github.com/matplotlib/matplotlib/issues/28045 ?

Reply:
Already on top of things. I like it.